Makefile

```diff
@@ -26,6 +26,7 @@ DOCKER_TAG=$(VERSION)
 GIT_SHA=$(shell git rev-parse HEAD)
 
 build:
+	go build ./command/health
```

Comment:
Need to do better on the makefile.  Sample diff:

```
-build:
-       go build ./command/health
+GO=CGO_ENABLED=0 GO111MODULE=on go
+GOFLAGS=-ldflags "-X github.com/edgexfoundry/edgex-go.Version=$(VERSION)"
+
+
+build: health
+
+.PHONY: health
+health:
+       $(GO) build $(GOFLAGS) -o $@ ./command/health
 
 docker: $(DOCKERS)
 
@@ -37,10 +44,11 @@ test:
 run:
        ./bin/edgex-consul-launch.sh
```

Issues:

- Need to set GO and GOFLAGS esp CGO_ENABLED=0 in order to produce a static binary.  As it is, it produces a libc-dependent binary which does not meet the requirements.
- The docker target needs to build the binary for inclusion in the container.  See Dockerfile coments.